addons/web/static/src/legacy/js/libs/bootstrap.js

```diff
@@ -79,6 +79,9 @@ $.fn.tooltip.Constructor.prototype.show = function () {
     // same time
     $('.tooltip').remove();
 
+    if (this.element.classList.contains('o_invisible_modifier')) {
```

Comment:
that's not what I had in mind, you can do the exact same fix you did earlier, but move it in this file.